src/phoenix/trace/semantic_conventions.py

```diff
@@ -57,10 +57,13 @@ class MimeType(Enum):
     JSON = "application/json"
 
 
+LLM_FUNCTION_CALL = "llm.function_call"
+LLM_INVOCATION_PARAMETERS = "llm.invocation_parameters"
+LLM_MESSAGES = "llm.messages"
+LLM_MODEL_NAME = "llm.model_name"
```

Comment:
can we add some doctrings above these? I think some of these are not default intuitive and worth prepping a bit for moving these into a spec.

Reply:
addressed in [f34a16c](https://github.com/Arize-ai/phoenix/pull/1135/commits/f34a16ca205f61455c30d2e8e4d1cfb8b64de41d)